addon/components/bs4/bs-navbar.js

```diff
@@ -4,7 +4,19 @@ import Navbar from 'ember-bootstrap/components/base/bs-navbar';
 export default Navbar.extend({
   classNameBindings: ['breakpointClass', 'backgroundClass'],
 
-  type: 'light',
+  type: computed('appliedType', {
+    get() {
+      return this.get('appliedType');
+    },
+
+    set(key, value) { // eslint-disable-line no-unused
+      let newValue = (!value || value === 'default') ? 'light' : value;
+      this.set('appliedType', newValue);
+      return newValue;
+    }
+  }),
+
+  appliedType: 'light',
```

Comment:
I might be missing something, but can't you just return a literal 'light' in the CP's getter, and remove the `this.set` call in the setter?